static/js/compose.js

```diff
@@ -810,120 +792,9 @@ exports.initialize = function () {
-        $("#compose-send-button").removeAttr("disabled");
-        switch (err) {
-        case 'BrowserNotSupported':
-            msg = i18n.t("File upload is not yet available for your browser.");
-            break;
-        case 'TooManyFiles':
-            msg = i18n.t("Unable to upload that many files at once.");
-            break;
-        case 'FileTooLarge':
-            // sanitization not needed as the file name is not potentially parsed as HTML, etc.
-            var context = { file_name: file.name };
-            msg = i18n.t('"__file_name__" was too large; the maximum file size is 25MiB.', context);
-            break;
-        case 'REQUEST ENTITY TOO LARGE':
-            msg = i18n.t("Sorry, the file was too large.");
-            break;
-        case 'QuotaExceeded':
-            msg = i18n.t("Upload would exceed your maximum quota."
-                      + " Consider deleting some previously uploaded files.");
-            break;
-        default:
-            msg = i18n.t("An unknown error occurred.");
-            break;
-        }
-        $("#error-msg").text(msg);
-    }
-
-    function uploadFinished(i, file, response) {
-        if (response.uri === undefined) {
-            return;
-        }
-        var textbox = $("#new_message_content");
-        var split_uri = response.uri.split("/");
-        var filename = split_uri[split_uri.length - 1];
-        // Urgh, yet another hack to make sure we're "composing"
-        // when text gets added into the composebox.
-        if (!compose_state.composing()) {
-            compose_actions.start('stream');
-        }
-
-        var uri = make_upload_absolute(response.uri);
-
-        if (i === -1) {
-            // This is a paste, so there's no filename. Show the image directly
-            textbox.val(textbox.val() + "[pasted image](" + uri + ") ");
-        } else {
-            // This is a dropped file, so make the filename a link to the image
-            textbox.val(textbox.val() + "[" + filename + "](" + uri + ")" + " ");
-        }
-        compose_ui.autosize_textarea();
-        $("#compose-send-button").removeAttr("disabled");
-        $("#send-status").removeClass("alert-info")
-                         .hide();
-
-        // In order to upload the same file twice in a row, we need to clear out
-        // the #file_input element, so that the next time we use the file dialog,
-        // an actual change event is fired.  This is extracted to a function
-        // to abstract away some IE hacks.
-        clear_out_file_list($("#file_input"));
-    }
-
-    // Expose the internal file upload functions to the desktop app,
-    // since the linux/windows QtWebkit based apps upload images
-    // directly to the server
-    if (window.bridge) {
-        exports.uploadStarted = uploadStarted;
-        exports.progressUpdated = progressUpdated;
-        exports.uploadError = uploadError;
-        exports.uploadFinished = uploadFinished;
-    }
```

Comment:
Deleting this will break the legacy desktop app.  https://github.com/zulip/zulip-desktop

Why did you delete it?  

When you're doing this sort of "move code" change, you should never make any functional changes in the same commit.  We're lucky I caught this, since it's pretty hard to notice when reviewing.